rplugin/python3/molten/moltenbuffer.py

```diff
@@ -146,14 +148,19 @@ def open_image_popup(self, silent=False) -> bool:
         for chunk in output.chunks:
             if isinstance(chunk, ImageOutputChunk):
                 try:
+                    if isinstance(self.canvas, WeztermCanvas):
+                        self.canvas.send_image(chunk.img_path)
+                        return True
+
```

Comment:
I don't think I like this this here. Ideally just opening the output window should show the image again if it's been closed.

The `open_image_popup` function should stick to opening images completely outside of the terminal.